.github/workflows/prod-test.yml

```diff
@@ -131,3 +131,10 @@ jobs:
       - name: Run Tests (standalone) (Linux and Windows)
         if: matrix.os != 'macos-latest'
         run: yarn test:dist-standalone --maxWorkers=2
+
+      # #8073 test
+      - name: Run Tests (PRETTIER_FALLBACK_RESOLVE)
```

Comment:
We need to change the order of the steps. The coverage report from this run is uploaded to Codecov.

Reply:
I thought they can merge.

Reply:
Seems like [they can](https://docs.codecov.io/docs/merging-reports) if you upload them separately: run test1, upload report1, run test2, upload report2.

Reply:
Sorry, I know codecov can, I mean jest should.

Reply:
No, it doesn't do that.